src/main/cpp/commands/drivetrain/DriveCommand.cpp

```diff
@@ -8,10 +8,14 @@ DriveCommand::DriveCommand(std::shared_ptr<Drivetrain> drivetrain,
     distance_ = distance;
     max_output_ = max_output;
 
-    this->AddRequirements(drivetrain_.get());
+    if (drivetrain_ != nullptr) {
+        this->AddRequirements(drivetrain_.get());
+    }
```

Comment:
This is a best practice move to ensure we don't ever try to do anything with nullptrs. I think we *should* do this for all commands, but also we shouldn't be sending nullptr to commands. We can't use OKC_CHECK or VOKC_CHECK though because those things return and this is a constructor